tron/utils/trontimespec.py

```diff
@@ -275,7 +275,8 @@ def handle_timezone(self, out, tzinfo):
                 for _ in range(24):
                     out += datetime.timedelta(minutes=60)
                     try:
-                        out = self.timezone.localize(out)
+                        if out.tzinfo is None:
+                            out = self.timezone.localize(out)
```

Comment:
This whole method is a bit weird (do we ever need to add more than 1 hour?) but in theory, we should use self.timezone.localize(out, is_dst=None) to force it to check again if the time exists and raise NonExistentTimeError if it still doesn't.

Reply:
well, i figured i'd rather add the minimal amount of code that implements the intended logic than change it and search for code paths that would break.

but i agree that adding 1 hour should always be enough, if you can give an advance shipit on such change - i'll merge that tomorrow.